packages/code-gen/test/e2e/openapi.test.js

```diff
@@ -169,6 +218,18 @@ test("code-gen/e2e/openapi", async (t) => {
     }
   });
 
+  t.test("assert on RouteExtensions", async (t) => {
+    const { paths } = JSON.parse(await readFile(outputFile, "utf-8"));
+
+    // find and destructure route with uniqueName: `GroupUpload`
+    const [, { post: groupUpload }] = Object.entries(paths).find(
+      ([path]) => path === "/group/file",
```

Comment:
nit: find by operationId

Reply:
Yes, would have been nice. But since the routes are declared by path strings. It is more "easy" to find on path.

```json
{
  '/{id}': {
    get: {
      operationId: 'ServerGetId',
      ...
    },
    get: {
      operationId: 'Server**',
      ...
    }    
  }
}
```